tools/build_defs/scala/scala.bzl

```diff
@@ -26,6 +26,40 @@ def _adjust_resources_path(path):
     return dir_1 + dir_2, rel_path
   return "", path
 
+def _build_nosrc_jar(ctx, buildijar):
+  res_cmd = ""
+  for f in ctx.files.resources:
+    c_dir, res_path = _adjust_resources_path(f.path)
+    change_dir = "-C " + c_dir if c_dir else ""
+    res_cmd = "\n{jar} uf {out} " + change_dir + " " + res_path
+  ijar_cmd = ""
+  if buildijar:
```

Comment:
yes, but not here:
https://github.com/bazelbuild/bazel/pull/948/files#diff-981b14fba970d9a9d0dd316467d216eeR52

where bazel it checking that an ijar war produced. There are two places where ijar identity is relevant;

1. where bazel checks that all outputs are created by some rule.
2. where dependant scala rules look for their compile time dependency.

So, in 2 above, I don't bother to give it the path to the copy, but I generate that path for 1 so bazel sees that it is created.

Would you prefer another approach?